### PR TITLE
New title component and some docs fixes and tests

### PR DESCRIPTION
**Pull request by hadasfa**

### CODE REVIEW

File: src/components/Dialog/Dialog.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ import { AnimationType, HideShowEvent } from "../../constants/dialog";
 import { VibeComponentProps } from "../../types";
 import * as PopperJS from "@popperjs/core";
 import styles from "./Dialog.module.scss";
+import { ComponentDefaultTestId, getTestId } from "../../tests/test-ids-utils";
```
Comment (hadasfa) [inline src/components/Dialog/Dialog.tsx:17]: for tooltip interaction tests

File: src/components/Tooltip/__tests__/tooltip.interactions.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { expect } from "@storybook/jest";
+import { Screen } from "@testing-library/react";
+import { userEvent, waitFor } from "@storybook/testing-library";
+import { ComponentDefaultTestId, getTestId } from "../../../tests/test-ids-utils";
+import { ONE_LINE_ELLIPSIS_TEST_ID } from "../../Text/__tests__/text-tests-constants";
+
+export async function isTestTooltipShowOnHover(
```
Comment (shlomitc) [inline src/components/Tooltip/__tests__/tooltip.interactions.ts:7]: This function is doing a multiple stuff
1. trigger show
2. assertion
3. trigger hide

so I think it's better to call it ("triggerTooltipAndAssert")

File: src/components/Typography/TypographyConstants.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+export enum TypographyColor {
+  PRIMARY = "primary",
+  SECONDARY = "secondary",
+  ON_PRIMARY = "onPrimary",
+  ON_INVERTED = "onInverted"
+}
+
+export enum TypographyAlign {
```
Comment (shlomitc) [inline src/components/Typography/TypographyConstants.ts:8]: I thought we decided to remove it

**Review by shlomitc — CHANGES_REQUESTED** [review]

File: src/components/Typography/TypographyConstants.ts (hunk 3 above)
Comment (hadasfa) [reply src/components/Typography/TypographyConstants.ts:8]: we not talked about it as far as I remember, why to remove it?

Comment (shlomitc) [reply src/components/Typography/TypographyConstants.ts:8]: in most cases, layout is related to containers and not the element itself.
for example - a table cell will control the alignment of its content, and not the textual content itself
in other cases - you would probably use flex/position to align content that also contains text

File: src/components/Tooltip/__tests__/tooltip.interactions.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  canvas: Screen,
+  getHoverableElement: () => Promise<HTMLElement>,
+  tooltipData: { id?: string; content?: string } = {}
+) {
+  // If we trying to get the element before it finishing it's first render we receive the instance without the tooltip wrapper and
+  // the userEvent.hover will not work (I did not find a way to implement this using react testing library or storybook interaction tests
+  await new Promise(resolve => {
```
[the remark this answers is not in the document]
Comment (hadasfa) [reply src/components/Tooltip/__tests__/tooltip.interactions.ts:14]: Did no found any other solution that worked (wait for did not work here) - what do you suggest

File: src/components/Tooltip/__tests__/tooltip.interactions.ts (hunk 2 above)
Comment (hadasfa) [reply src/components/Tooltip/__tests__/tooltip.interactions.ts:7]: Assert is not clear enough... I can change to testHoverTooltipTrigger?

**Review by shlomitc — APPROVED** [review]
just try to improve the test and lets go

**Review by shlomitc — APPROVED** [review]

**Pull request merged**